src/popup/pages/SaveMnemonic/index.tsx

```diff
@@ -57,8 +57,8 @@ class SaveMnemonic extends Component<WithStyles & IProps, {}> {
   }
 
   private createWallet = (saveFile: boolean) => {
-    const { history, store: { saveMnemonicStore } }: any = this.props;
-    history.push('/loading');
+    const { store: { saveMnemonicStore, routerStore } }: any = this.props;
+    routerStore.push('/loading');
```

Comment:
move the routing logic to the store. lets keep UI logic only in the components and biz logic in the stores.